web/src/api/auth/login-callback.ts

```diff
@@ -54,11 +54,10 @@ export const auth0Login = withApiAuthRequired(
 
     if (isEmailUser(auth0User)) {
       try {
-        const userData = await FetchUserByAuth0IdSdk(client).FetchUserByAuth0Id(
-          {
-            auth0Id: auth0User.sub,
-          }
-        );
+        const userData = await FetchUserByAuth0IdSdk(client).FetchEmailUser({
```

Comment:
this can lead to edge cases of e.g. returning two different users. best to have 2 GraphQL queries (can be executed in the same request), one for email, one for auth0Id, prioritize the auth0Id and only use email as fallback

Reply:
Updated